### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by jburel**
Replace travis by GHA
Update README since we now only support BF formulae

see https://github.com/jburel/homebrew-alt/actions/runs/425784647

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    name: Install Bio-Formats
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up
+        run: |
+          brew update
+          brew tap ome/alt file://$(pwd)
+      - name: Install Bio-Formats Formula
+        run: brew install bioformats
```
Comment (dgault) [inline .github/workflows/main.yml:20]:
```suggestion
        run: | 
          brew install bioformats
          brew test bioformats
```

**Review by dgault — COMMENTED** [review]
Action looks good and runs as expected, only thing I see missing is the `brew test bioformats` command which was being run with travis

**Comment by jburel** [comment]
with the last commit https://github.com/jburel/homebrew-alt/actions/runs/428458302 is green
@dgault ok to merge?

**Comment by dgault** [comment]
Good to merge

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # OME Homebrew Tap
 
-This repository is a [Homebrew tap](https://github.com/Homebrew/brew/blob/master/docs/brew-tap.md) which contains formulae for [OME](https://www.openmicroscopy.org/) software.
+[![Actions Status](https://github.com/ome/homebrew-alt/workflows/Build/badge.svg)](https://github.com/ome/homebrew-alt/actions)
+
+This repository is a [Homebrew tap](https://github.com/Homebrew/brew/blob/master/docs/brew-tap.md) which contains formulae for [Bio-Formats](https://www.openmicroscopy.org/bio-formats/) software.
```
Comment (joshmoore) [inline README.md:5]: Maybe one day we'll need to switch this back to OME :smile:

**Pull request merged**